grid-demo/src/app/angular/column-ordering/column-ordering.component.ts

```diff
@@ -1,7 +1,27 @@
+import { CdkDragDrop } from '@angular/cdk/drag-drop';
 import { Component } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import { Vm, generateVms, columns } from './../../data/vm.generator';
 
 @Component({
   templateUrl: './column-ordering.component.html',
   styleUrls: ['./column-ordering.component.scss'],
 })
-export class AngularColumnOrderingComponent {}
+export class AngularColumnOrderingComponent {
+  readonly vms: Observable<Vm[]>;
+  columns = new BehaviorSubject<{ field: keyof Vm; displayName: string }[]>(columns);
```

Comment:
This property could be marked `readonly`.

Reply:
fixed